components/actionmenu/stories/actionmenu.stories.js

```diff
@@ -1,41 +1,58 @@
+import { ArgGrid, Container } from "@spectrum-css/preview/decorators/utilities.js";
+import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { isOpen } from "@spectrum-css/preview/types";
+
 import { default as ActionButton } from "@spectrum-css/actionbutton/stories/actionbutton.stories.js";
-import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
 import { default as Menu } from "@spectrum-css/menu/stories/menu.stories.js";
 import { default as Popover } from "@spectrum-css/popover/stories/popover.stories.js";
-import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isOpen } from "@spectrum-css/preview/types";
-import packageJson from "../package.json";
+
+import { Template as IconTemplate } from "@spectrum-css/icon/stories/template.js";
 import { ActionMenuGroup } from "./actionmenu.test.js";
+import { Template } from "./template.js";
+
+import metadata from "../dist/metadata.json";
+import packageJson from "../package.json";
 
 /**
- * The action menu component is an action button with a popover. The `is-selected` class should be applied to the button when the menu is open. Note that the accessibility roles are different for an action menu compared to a normal menu.
+ * Action menu allows users to access and execute various commands or tasks related to their current workflow. It's typically triggered from an action button by user interaction.
+ *
+ * Note that the accessibility roles are different for an action menu compared to a normal menu. The action menu is a combination of a menu, popover, and action button.
  */
 export default {
 	title: "Action menu",
 	component: "ActionMenu",
 	argTypes: {
-		withTip: Popover.argTypes.withTip,
-		position: Popover.argTypes.position,
-		isOpen,
-		iconName: {
-			...(IconStories?.argTypes?.iconName ?? {}),
-			if: false,
+		position: {
+			...Popover.argTypes.position,
+			options: [
+				"bottom-start",
+				"bottom-end",
+				"start-top",
+				"end-top",
+			]
 		},
-		label: {
-			name: "Label",
-			type: { name: "string" },
+		isOpen,
+		hasLongPress: {
+			name: "Long press",
+			description: "If the trigger supports a long-press action which triggers the menu, this should be set to true.",
+			type: { name: "boolean" },
 			table: {
-				type: { summary: "string" },
-				category: "Content",
+				type: { summary: "boolean" },
+				category: "Accessibility",
 			},
-			control: { type: "text" },
+			control: "boolean",
 		},
-		items: { table: { disable: true } },
+		menuArgs: { table: { disable: true } },
+		triggerArgs: { table: { disable: true } },
```

Comment:
I go back and forth on the removal of the `iconName` control from the table. We have a story that demonstrates that users have the ability to change the icon, but then removed the control so users can't experiment, so I'm split. 

Anyways, I don't think it's blocking this PR from merging, but I did want to share some thoughts.